strongbox-aql/src/main/twig/AQL.g4.twig

```diff
@@ -127,60 +144,92 @@ fragment W : [wW];
 fragment X : [xX];
 fragment Y : [yY];
 fragment Z : [zZ];
+#}
+
+{% for layout, layoutDesc in layoutMap -%}
+    {{ 'LAYOUT_' ~ (layoutDesc.layoutAlias | upper) }}:  
+
+    '{{layoutDesc.layoutAlias}}'
+
+{#
+    {% for l in split(layoutDesc.layoutAlias, '') -%}
+        {{ (l | upper) ~ ' ' }}
+    {%- endfor %}
+#}
+
+    {{ ';\n\n' }}
+{%- endfor %}
+
+{% for layout, layoutDesc in layoutMap -%}
+    {% for layoutCoordinate in layoutDesc.artifactCoordinates -%}
+    
+        {{ 'LAYOUT_COORDINATE_' ~ (layout | upper) ~ "_" ~ (layoutCoordinate | upper) }}:  
+        
+        '{{ layoutCoordinate }}'
+
+{#
+        {% for l in split(layoutCoordinate, '') -%}
+            {{ (l | upper) ~ ' ' }}
+        {%- endfor %}
+#}
+
+        {{ ';\n\n' }}
+    {%- endfor %}
+{%- endfor %}
 
 STORAGE
 :
-    S T O R A G E
```

Comment:
Does this no longer work like this?

Reply:
we temporary dropped case insensitive feature, see [SB-1142](https://youtrack.carlspring.org/issue/SB-1142)